src/commands/Utility/quote.ts

```diff
@@ -142,6 +149,34 @@ async function addQuote(interaction: Discord.CommandInteraction, collection: Col
 		author = interaction.options.getUser("author", true),
 		url = interaction.options.getString("url", false)
 
+	let pictureUrl = interaction.options.getString("pictureUrl", false) ?? undefined
+
+	const urlSplit = url?.split("/")
+
+
+	if (urlSplit) {
+		if (urlSplit.length === 7) {
```

Comment:
urlSplit can be undefined, idk why ts didn't pick up on that

Reply:
thats why i do a if (urlSplit) ????

Reply:
Ignore lol